torch/nn/parallel/data_parallel.py

```diff
@@ -129,14 +131,14 @@ def __init__(self, module, device_ids=None, output_device=None, dim=0):
 
         self.dim = dim
         self.module = module
-        self.device_ids = list(map(lambda x: _get_device_index(x, True), device_ids))
-        self.output_device = _get_device_index(output_device, True)
-        self.src_device_obj = torch.device("cuda:{}".format(self.device_ids[0]))
+        self.device_ids = device_ids
```

Comment:
does it mean the behavior here changed a bit as it no longer checks whether provided strings are valid? The change looks OK to me, as no BC issue here.